lib/lightning_web/channels/attempt_channel.ex

```diff
@@ -64,17 +64,27 @@ defmodule LightningWeb.AttemptChannel do
   end
 
   def handle_in("attempt:complete", payload, socket) do
-    %{
-      "reason" => reason,
-      "error_type" => error_type,
-      "error_message" => error_message
-    } =
+    params =
       payload
+      |> Map.get_and_update("reason", fn _ -> :pop end)
```

Comment:
:metal:

Reply:
only the `_` if you may ... in the future if we want to switch to `mix credo --strict` we won't to have to fix all these entries.